mx.py

```diff
@@ -581,6 +581,7 @@ def __init__(self, parents=None):
                                 "projects and store it in the given <file>. If <file> is 'default', the compilation database will "
                                 "be stored in the parent directory of the repository containing the primary suite. This option "
                                 "can also be configured using the MX_COMPDB environment variable. Use --compdb none to disable.")
+        self.add_argument('--arch', action='store', dest='arch', help='force use the specified architecture', choices=['aarch64', 'amd64', 'sparcv9'])
```

Comment:
Why allow sparcv9? Do we still need that for some compatibility?

Reply:
`get_arch` could return that, so that's why I put it in the allowed choices.